ci-operator/step-registry/acm/fetch-managed-clusters/acm-fetch-managed-clusters-commands.sh

```diff
@@ -16,7 +16,11 @@ export HUB_OCP_PASSWORD
 
 if [ -s /tmp/ci/managed.cluster.name ] && ! grep -q "null" /tmp/ci/managed.cluster.name; then
     echo "managed.cluster.name file was found saving as artifact."
-    cp -r /tmp/ci/managed.cluster.name $ARTIFACT_DIR/
+    cp -r /tmp/ci/managed.cluster.name $SHARED_DIR/
```

Comment:
Not really urgent, you couple probably just do `cp -r /temp/ci/managed.cluster.* $SHARED_DIR/`, unless it copies a bunch of unneeded files.

Reply:
We've decided to push this pr through without this change. May make the change in the future.